.gitignore

```diff
@@ -26,7 +26,7 @@ a.out
 /lib/version.py
 /lib/version.sh
 /lib/version.txt
-/test/build/61_force-auto.bats
+/test/61_force-auto.bats
```

Comment:
The number prefix is for ordering within a directory. Because it's a one-off now, the prefix should be removed.